tests/test_watchmedo.py

```diff
@@ -48,3 +48,28 @@ def test_load_config_invalid(tmpdir):
         watchmedo.load_config(yaml_file)
 
     assert not os.path.exists(critical_dir)
+
+
+def make_dummy_script(tmpdir, n=10):
+    script = os.path.join(tmpdir, 'auto-test-%d.py' % n)
+    with open(script, 'w') as f:
+        f.write('import time\nfor i in range(%d):\n\tprint("+++++ %%d" %% i)\n\ttime.sleep(1)\n' % n)
```

Comment:
Maybe an idea for Windows test failures:
```suggestion
        f.write('import time\nfor i in range(%d):\n\tprint("+++++ %%d" %% i, flush=True)\n\ttime.sleep(1)\n' % n)
```